docs/produto/acceptance-test.md

```diff
@@ -0,0 +1,70 @@
+# Introdução
+Testes de aceitação são um dos principais tipos de testes usados para validar o desenvolvimento de requisitos, que são construídos ao longo do processo de percepção do produto. O resultado desses testes devem demonstrar que o sistema é capaz de executar as funcionalidades existentes de forma adequada e coerente em termos de tempo de resposta e validade dos resultados, já a construção desses testes se dá a partir de situações de uso do próprio sistema. Além disso, os testes de aceitação podem  também avaliar a aplicação quanto à navegabilidade, usabilidade e acessibilidade, nesse sentido serão propostas algumas tarefas e, durante a sua realização, os avaliadores que compoẽm o time deverão atentar-se à identificação de problemas ou dificuldades do usuário ao navegar no app. O intuito deste documento é relatar desde os objetivos a quais serão os testes aplicados e como funcionarão os critérios de sucesso.        
+
+# Objetivo
+O principal objetivo da realização desses testes é verificar aspectos citados anteriormente, como acessibilidade, navegabilidade e usabilidade. Para isso, pautamos nossos objetivos de forma que os problemas relacionados a esses aspectos, caso existam, sejam evidenciados durante as avaliações. As grandes áreas que são compreendidas nessas verificações são:          
+* **Precisão**: Quantos erros a pessoa cometeu? E eles foram fatais ou a pessoa conseguiu se recuperar com as informações recebidas pelo sistema?              
+* **Desempenho**: Quanto tempo e quantos passos são necessários para que a pessoa complete tarefas básicas?       
+* **Lembrança**: O quanto a pessoa se lembra mais tarde ou depois de períodos sem usar?           
+* **Resposta Emocional**: Como a pessoa se sentiu depois de completar a tarefa? A pessoa estava confiante ou estressada?        
+
+O entendimento do usuário em relação às funcionalidades, à facilidade ou não em encontrar funcionalidades de acordo com a disposição na tela e a navegação, do login à execução das principais funcionalidades, serão as pontos principais de observação. Para padronizar esse processo, serão determinadas quais informações serão passadas ao usuário e quais os fluxos de teste.         
+
+# Usuários de Teste
+Para que fosse possível o desenvolvimento da aplicação mesmo que sem um cliente real, foram criados perfis diversificados de usuários fictícios utilizando a técnica de [personas](https://github.com/fga-eps-mds/2020.1-stay-safe-docs/blob/master/docs/produto/product-definition.md). Para realizar os testes de aceitação é pretendido que a equipe encontre pessoas que possuam um perfil semelhante ao das personas, sendo estimada a participação de pelo menos cinco usuários e que o ambiente onde acontecerá os testes seja informal, havendo o mínimo de intervenções externas possíveis durante a realização dos mesmos.
```

Comment:
I would be better if the personas' document link was the one of the official documentation.